SSOAR.js

```diff
@@ -273,6 +272,7 @@ var testCases = [
 		"items": [
 			{
 				"itemType": "journalArticle",
+				"title": "Book review of \"Transatlantic Feminisms in the Age of Revolutions\" by Lisa L. Moore, Joanna Brooks and Caroline Wigginton (Eds.)",
```

Comment:
The quotes changed here. Everything else in the test cases should be just reordering.